tests/files_test_partisn/partisn_1D_expected.inp

```diff
@@ -30,7 +30,7 @@ t
 / 
 / Note: NAMES is not all inclusive. Only NAMES that are present in
 / meshed area are listed.
-names= He4 Hg202 Hg200 Hg201 Hg196 Hg204 He3 Hg198 Hg199 
+names= He3 He4 Hg196 Hg198 Hg199 Hg200 Hg201 Hg202 Hg204 
```

Comment:
Why the changes to these test results?

Reply:
Because the output sequence of python2 and python3 are different.
There is a set storing these nuclide names, we iterate the set and print out the nucldie names.
However, the default iterate sequence for this set seems to be different in python2 and python3.

You can see that their content is actually the same, but just with different sequence. Well, the sequence is not important.
But for the compatibility, we need to sort and then output. The new results are sorted output, the old results not sorted.

Reply:
Gotcha. That makes sense and make the output more understandable